pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -588,6 +574,47 @@ func convertToUsers(guestInfo *libvirt.DomainGuestInfo) []api.User {
 	return users
 }
 
+func convertToDisks(guestInfo *libvirt.DomainGuestInfo, fileSystem libvirt.DomainGuestInfoFileSystem) []api.FSDisk {
+	var disks []api.FSDisk
```

Comment:
Please define variable close to where it is used. The code is then easier to read.